docs_2.0/manual_EN/3.quick-start/4.nebua-graph-crud.md

```diff
@@ -0,0 +1,474 @@
+# Nebula Graph CRUD
+
+This topic describes the basic CRUD operations in Nebula Graph.
+
+## Graph space and Nebula Graph schema
+
+A Nebula Graph instance consists of one or more graph spaces. Graph spaces are physically isolated from each other. You can use different graph spaces in the same instance to store different data sets.
+
+![Nebula Graph and graph spaces](https://docs-cdn.nebula-graph.com.cn/docs_2.0/3.quick-start/nebula_graph_instance_and_graph_spaces.png)
+
+Before inserting data into a graph space, you have to define the database schema. Nebula Graph schema is based on the following components.
```

Comment:
To insert...?

Reply:
Both will do. If we use to insert, then we need to add a "first" at the end.